.github/workflows/light-client.yaml

```diff
@@ -56,6 +56,24 @@ jobs:
             cargo build -p ibc-client-starknet-cw --target wasm32-unknown-unknown --release
           cosmwasm-check target/wasm32-unknown-unknown/release/ibc_client_starknet_cw.wasm
 
+  starknet-storage-verifier-tests:
+    name: Run Starknet Storage Verifier tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+          workspaces: |
+            light-client
+
+      - name: Run tests
+        working-directory: ./light-client
+        run: |
+          cargo test --package starknet-storage-verifier --test storage_proof_tests --  --show-output
```

Comment:
I just realized we didn't have tests running `light-client` workspace. You can just use `cargo test` to run tests on the full workspace.

Reply:
Done in 0225a48995898fcd04f15041c51b53568a31689f